pkg/ovs/ovn-nb-logical_switch_port.go

```diff
@@ -25,6 +25,21 @@ func (c *OVNNbClient) CreateLogicalSwitchPort(lsName, lspName, ip, mac, podName,
 
 	// ignore
 	if exist {
+		if portSecurity && len(securityGroups) != 0 {
+			lsp, err := c.GetLogicalSwitchPort(lspName, true)
+			if err != nil {
+				klog.Error(err)
```

Comment:
log more info, E.g. what are you doing, and then err happened. Like following logging info

Reply:
And then return error

Reply:
The code has been updated to another style.